src/pip/_internal/commands/install.py

```diff
@@ -262,13 +259,6 @@ def run(self, options, args):
         if options.target_dir:
             options.ignore_installed = True
             options.target_dir = os.path.abspath(options.target_dir)
```

Comment:
And maybe here:
```suggestion
            options.target_dir = os.path.abspath(options.target_dir)
            assert not os.path.exists(options.target_dir) or os.path.isdir(options.target_dir), "Target path exists but is not a directory, will not continue."	
```

Reply:
I feel that this will just cause confusion to future readers, so I added a comment above `decide_user_install` instead.

Reply:
The purpose of comments is different from `assert`s. But if nobody wants them, so be it.